release-notes.md

```diff
@@ -1,5 +1,16 @@
 # Release notes
 
+## June 17, 2019
+
+The following updates were made in this release:
+
+**iOS Places 1.1.0**
+
+* Added a new API to return an error code when there is a failure retrieving nearby places.
+* When privacy status changes to opt-out, all Places-related data will now be wiped from the device.
+* Fixed a bug sometimes causing Places events following a first launch to be lost due to bad network conditions.
+* Fixed an issue where token replacement via Rules Engine would sometimes reference the incorrect POI when processing POIs in quick succession
```

Comment:
Please reword so that it reads as follows:

"Fixed an issue where, when processing POIs in quick succession, token replacement via the Rules Engine sometimes referenced the incorrect POI."